### ISSUE TITLE
Progress?

### ISSUE THREAD

**Issue by haf** [pocketberserker/FsAttoparsec#10]
Hi there,

I would be great to have a high-performance binary parser in F#. Are you considering maintaining and moving forward with this project?

**Comment by pocketberserker** [comment]
I think that I want to support .NET Core. However, I have not taken time to work.
If you think that you want to grow FsAttoparsec, I will add you to GitHub Collaborators and nuget Owners.

**Comment by haf** [comment]
Maybe. I'm going through the code right now, trying to make sense of it. Let's see if I can improve anything.

**Comment by haf** [comment]
@pocketberserker Yes, please give me access to the NuGet. We'll be using this library for binary parsing at Qv2.

**Comment by wallymathieu** [comment]
I've done some .net core ports, so I can look over it as well.

**Comment by pocketberserker** [comment]
@haf I added you to onwer of [FsAttoparsec](https://www.nuget.org/packages/FsAttoparsec/) and invited collaborator of this repository.
Please check your e-mail.

**Comment by haf** [comment]
Thank you! I saw that you starred https://github.com/inhabitedtype/angstrom — seems like a good library to compare this implementation to!

This thread could be a good place to start coordinating what would need doing.

**Comment by haf** [comment]
I think a good first aim could be to parse HTTP so that Suave and Fable can use the data object model produced. https://gist.github.com/haf/9ff8742c0caeaaf23859ce80cc6b9b58
